_posts/2017/2017-07-19-reunification.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Reunification"
+layout: "post"
```

Comment:
All of the items under the _posts directory will default to `layout: "post"` so that tag won't be needed. Though place add a `description` tag.